addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -361,14 +361,17 @@ export class HtmlField extends Component {
     async commitChanges({ urgent } = {}) {
         if (this._isDirty() || urgent) {
             if (urgent) {
+                if (this.props.isInlineStyle) {
+                    this._toInline();
+                }
```

Comment:
Wait, didn't you check this already to not call `commitChanges` ?

Reply:
Only on during blur, not for other commit change requester.
And since I modified the _toInline function to be synchronous, we can now call it during urgent saves.